x-pack/plugins/observability_solution/apm/server/lib/helpers/get_apm_data_access_services.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ApmDataAccessServices, APMEventClient } from '@kbn/apm-data-access-plugin/server';
+import { MinimalAPMRouteHandlerResources } from '../../routes/apm_routes/register_apm_server_routes';
+
+export async function getApmDataAccessServices({
+  apmEventClient,
+  plugins,
+}: {
+  apmEventClient: APMEventClient;
+} & Pick<MinimalAPMRouteHandlerResources, 'plugins'>): Promise<ApmDataAccessServices> {
```

Comment:
Just a nit regarding the naming consistency - I see some places using `APM` and other `Apm` so we can decide which one to use and stick to it

Reply:
Yeah, this bothers me too. Something to address in the future.